en/guide/follow_me.md

```diff
@@ -0,0 +1,75 @@
+
+The API is used to supply the position(s) for the [target](../api_reference/structdronecore_1_1_follow_me_1_1_target_location.md) and the relative [follow position](../api_reference/structdronecore_1_1_follow_me_1_1_config.md) of the vehicle. Applications must get target position information from the underlying platform (or some other source). The location APIs for supported platforms are listed below:
+- Android: [Location](https://developer.android.com/reference/android/location/Location.html)
+- Apple: [Core Location Framework](https://developer.apple.com/documentation/corelocation)
+- Windows: [Windows.Devices.Geolocation](https://docs.microsoft.com/en-us/uwp/api/Windows.Devices.Geolocation)
+
+
+## Set the Follow Configuration
+
+By default the vehicle will follow directly behind the target at a height and distance of 8 metres. 
+You can (optionally) call [set_config()](../api_reference/classdronecore_1_1_follow_me.md#classdronecore_1_1_follow_me_1a4b92c3a042911dd9bdb378c686458a34) at any time to specify a different height, follow distance, relative position (front left/right/centre or behind) and responsiveness to target movements. 
+
+The code fragment below shows how to set the configuration:
+```cpp
+// configure follow me behaviour
+FollowMe::Config config;
+config.min_height_m = 12.f;  // Minimum height
+config.follow_dist_m = 20.f;  // Follow distance
+config.responsiveness = 0.2f;  // Higher responsiveness
+config.follow_direction = FollowMe::Config::FollowDirection::FRONT;  //Follow from front-centre
+
+// Apply configuration
+bool configured = device.follow_me().set_config(config);
+if (configured) {
+    // handle config failure
+    std::cout << "Configuration failed" << std::endl;
+}
+```
+
+The [get_config()](../api_reference/classdronecore_1_1_follow_me.md#classdronecore_1_1_follow_me_1a054aebafe0839a1028f277285b769fe5) method is provided to get the current configuration:
+```cpp
+auto curr_config = device.follow_me().get_config();
+```
+
+## Following a Target
+
+To start and stop following a target, call [start()](../api_reference/classdronecore_1_1_follow_me.md#classdronecore_1_1_follow_me_1a694749d43d527f85584df25a49b05ccf) and [stop()](../api_reference/classdronecore_1_1_follow_me.md#classdronecore_1_1_follow_me_1a6394507b0fb96bceebe6efd17f0529ce), respectively.
+
+In between the calls, use [set_curr_target_location()](../api_reference/classdronecore_1_1_follow_me.md#classdronecore_1_1_follow_me_1afb8c24ed93421e904b0f528569c7699a) to set the target position for the vehicle to follow. 
+DroneCore automatically resends the last set position at the rate required by the autopilot, but this can be overridden by setting the location at another rate.

```

Comment:
@shakthi-prashanth-m Is this correct? I assume so, but want to make sure!

Reply:
Yes. but rate remains same. I meant this **DroneCore keeps sending last set position at rate 1Hz until client app overrides by setting new location.**

Reply:
Thanks. I'll fix that text.